docs/table-epidemiology.md

```diff
@@ -6,8 +6,8 @@ Information related to the COVID-19 infections for each date-region pair.
 
 ## URL
 This table can be found at the following URLs depending on the choice of format:
-* [epidemiology.csv](https://storage.googleapis.com/covid19-open-data/v2/epidemiology.csv)
-* [epidemiology.json](https://storage.googleapis.com/covid19-open-data/v2/epidemiology.json)
+* [epidemiology.csv](https://storage.googleapis.com/covid19-open-data/v3/epidemiology.csv)
+* [epidemiology.json](https://storage.googleapis.com/covid19-open-data/v3/epidemiology.json)
 
 
 ## Schema
```

Comment:
Missing updates to the schema which were part of the v2 -> v3 migration (e.g. `total_` -> `cumulative_`). Other tables might be affected too, like hospitalizations and vaccinations.

Reply:
Thanks @a27cheung! It looks like you have only updated the schema docs for the epidemiology table, there are several others that need updating as well — the details are in the migration doc that you wrote a few months ago.

Reply:
Np! Haven't forgotten; just prefer many small changes. :)